doc: refactor platform plugin versioning (#1314)
apache/cordova-docs: www/docs/en/dev/platform_plugin_versioning_ref/index.md

}
```

The `--nosave` flag prevents adding and deleting the specified platform from the `package.json` file.

Example:

```bash
cordova platform add <platform[@<version>] | directory | git_url> --nosave
```

Here are various ways to add platforms.

*Adding with Cordova resolved name:*

Example:

```bash
cordova platform add android
```

The avaialble Cordova Resolved names are:

* `android`
* `electron`
* `ios`
* `browser`

> The `cordova-android` platform will be fetched from npm, added to the project and updates the `package.json` file.

Reviewer: ```suggestion
The latest version of the `cordova-android` platform will be fetched from npm, added to the project and the `package.json` file will be updated.
```

`updates` here and the other paragraphs sounds like strange English to me but I'm not a native speaker.

*Adding with Cordova resolved name and pinned version:*

Example:

```bash
cordova platform add android@7.1.4
```

> The `cordova-android` platform at version `7.1.4` will be fetched npm, added to the project and updates the `package.json` file.

Reviewer: ```suggestion
The `cordova-android` platform at version `7.1.4` will be fetched from npm, added to the project and updates the `package.json` file.
```

> The `cordova-android` platform at version `7.1.4` will be fetched from npm, added to the project and updates the `package.json` file.

Reviewer: ```suggestion
The `cordova-android` platform at version `7.1.4` will be fetched from npm, added to the project and the `package.json` file will be updaed.
```

*Adding with npm package name:*

Example:

```bash
cordova platform add cordova-android
```

> The `cordova-android` platform will be fetched from npm, added to the project and updates the `package.json` file.

Reviewer: ```suggestion
The `cordova-android` platform will be fetched from npm, added to the project and the `package.json` file will be updated.
```
